code/modules/ordeals/green_ordeals.dm

```diff
@@ -21,26 +22,25 @@
 	end_sound = 'sound/effects/ordeals/green_end.ogg'
 	level = 2
 	reward_percent = 0.15
-	spawn_places = 3
+	spawn_places = 6
 	spawn_amount = 1
 	spawn_type = /mob/living/simple_animal/hostile/ordeal/green_bot_big
 	place_player_multiplicator = 0.08
 	spawn_player_multiplicator = 0
 	color = COLOR_DARK_LIME
 
 // Dusk
-/datum/ordeal/simplecommander/green_dusk
+/datum/ordeal/simplespawn/green_dusk
 	name = "Dusk of Green"
 	annonce_text = "We constructed a looming tower to return whence we came."
 	level = 3
 	reward_percent = 0.2
 	annonce_sound = 'sound/effects/ordeals/green_start.ogg'
 	end_sound = 'sound/effects/ordeals/green_end.ogg'
 	color = COLOR_DARK_LIME
-	bosstype = list(/mob/living/simple_animal/hostile/ordeal/green_dusk)
-	grunttype = list(/mob/living/simple_animal/hostile/ordeal/green_bot)
-	bossnumber = 3
-	gruntnumber = 1
+	spawn_places = 3
+	spawn_amount = 1
+	spawn_type = /mob/living/simple_animal/hostile/ordeal/green_dusk
 
```

Comment:
currently the factories seem to scale with population, consider adding in 
	place_player_multiplicator = 0
	spawn_player_multiplicator = 0